src/lessons/Basics/03.vue

```diff
@@ -37,7 +38,7 @@ const validate = async (result, ipfs) => {
   }
 
   if (result.value === 1 && result.remainderPath === '') {
-    return {success: 'Great job!'}
+    return {success: "Great job! You've completed this series of lessons!"}
```

Comment:
minor, but we're using singlequotes `'` for strings in this project. The bigger issue is we should have CI checks and code linting as part of the PR process, but that's an issue for another PR!